pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2518,6 +2549,35 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 			// We assume AddOrUpdateIoBundle will preserve any
 			// existing IsPort/IsPCIBack/UsedByUUID
 			aa.AddOrUpdateIoBundle(log, *ib)
+
+			// Adding PF before VF to have correct boot order
+			if ib.Type == types.IoNetEthPF {
+				vfCnt, err := vfCount(phyAdapter)
+				if err != nil {
+					log.Functionf("Failed to get vfCount for phyAdapter %s", err)
+				}
+				if vfCnt > 0 {
+					err, vfs := createAndGetVF(ib.Ifname, vfCnt)
+					if vfs == nil {
+						log.Fatal("Failed to create VF for iface", ib.Ifname)
+					}
+					if err != nil {
+						log.Functionf("Error occured during obtaining info %s", err)
```

Comment:
Make the error string include vfCnt and set it in ib.Error (and set ib.ErrorTIme) and then do a log.Errorf which includes ib.Ifname and ib.Error

Reply:
Done